modules/localization/msf/msf_localization.cc

```diff
@@ -16,6 +16,8 @@
 
 #include "modules/localization/msf/msf_localization.h"
 
+#include <boost/property_tree/ptree.hpp>
```

Comment:
let's use protocol buffer text version of config. one example: https://github.com/ApolloAuto/apollo/blob/master/modules/localization/conf/localization_config.pb.txt